syntax = "proto3";

message OrderOperationRequest {
    int32 user_id = 1;
    int32 product_id = 2;
}

message OrderOperationResponse {
    int32 status_code = 1;
}

service OrderManagement {
    rpc add_product (OrderOperationRequest) returns (OrderOperationResponse);
    rpc delete_product (OrderOperationRequest) returns (OrderOperationResponse);
}